### PR TITLE
[14.0][FIX] stock_request_analytic: Adding the correct groups to the views to avoid permissions errors

### PR DESCRIPTION
**Pull request by victoralmau**
Adding the correct groups to the views to avoid permissions errors

@Tecnativa

### CODE REVIEW

File: stock_request_analytic/views/stock_request_order_views.xml [hunk 1]
```diff
@@ -8,8 +8,16 @@
         <field name="inherit_id" ref="stock_request.stock_request_order_form" />
         <field name="arch" type="xml">
             <div name="button_box" position="inside">
-                <field name="analytic_account_ids" invisible="1" />
-                <field name="analytic_tag_ids" invisible="1" />
+                <field
+                    name="analytic_account_ids"
+                    groups="analytic.group_analytic_accounting"
```
Comment (pedrobaeza) [inline stock_request_analytic/views/stock_request_order_views.xml]: Putting groups here in this version doesn't avoid permission errors. It should be at view level. Anyway, removing the analytic group is only for not showing the fields, not for permissions.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The changes on analytic seems weird

File: stock_request_analytic/views/stock_request_order_views.xml (hunk 1 above)
Comment (victoralmau) [reply stock_request_analytic/views/stock_request_order_views.xml]: Ok, it is already changed and the corresponding groups are set in the views.

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**